internal/pkg/template/templates/workloads/services/rd-web/cf.yml

```diff
@@ -84,7 +84,7 @@ Resources:
                 Value: !Ref EnvName
               - Name: COPILOT_SERVICE_NAME
                 Value: !Ref WorkloadName
-{{- if eq .Network.SubnetsType "PrivateSubnets"}}
+{{- if or .Network.SubnetIDs (ne .Network.SubnetsType "")}}
```

Comment:
Gotcha that makes sense 💭 
My goal was to reduce the duplicate conditionals in the template as much as possible - maybe that means instead a method in `WorkloadOpts` like this:

```go
func (opt WorkloadOpts) RequiresVPCConnector() bool {
  if opt.WorkloadType != "Request-Driven Web Service" { return false }
  return len(opt.Network.SubnetIDs) > 0 || opt.Network.SubnetsType != ""
}
```
This isn't too bad - I just wish we had specific Opts structs instead of a generic `WorkloadOpts`. I'll let you decide!